_source/user-guide/integrations/endpoints.md

```diff
@@ -13,23 +13,30 @@ contributors:
 Notification endpoints can receive notifications when an alert is triggered,
 when a user shares a Kibana object,
 or when Logz.io finds new insights in your logs.
+
+
 Logz.io supports sending notifications to
-BigPanda, Datadog, PagerDuty, Slack, VictorOps, or your own custom endpoint.
+Opsgenie, BigPanda, Datadog, PagerDuty, Slack, VictorOps, or your own custom endpoint.
+
+![Notification endpoints](https://dytvr9ot2sszz.cloudfront.net/logz-docs/notification-endpoints/notification-endpoints.png)
 
-![Notification endpoints](https://dytvr9ot2sszz.cloudfront.net/logz-docs/alerts/alerts--alert-endpoints.png)
 
-You can manage your endpoints
-from [**Alerts & events > Notification endpoints**](https://app.logz.io/#/dashboard/alerts/endpoints).
-You can also add new endpoints when you're configuring a new notification.
 
 #### To manage notification endpoints
 
-* To add a new endpoint,
-  click **Add endpoint**,
-  fill in the required information for your endpoint,
+You can manage your endpoints
+from [**Alerts & events > Alert endpoints**](https://app.logz.io/#/dashboard/alerts/endpoints).
+You can also add new endpoints when you're configuring a new notification.
+
+* To add a new endpoint, click **Add endpoint**,
+  fill in the required information,
   and click **Save**.
-  You can do this from the [_Notification endpoints_](https://app.logz.io/#/dashboard/alerts/endpoints) page,
-  or anywhere in Logz.io where you choose the recipient of a message or notification.
+
+  For example, to add an Opsgenie endpoint, you'll need to select **Opsgenie** from the dropdown list, name the endpoint, and fill in your Opsgenie API key. 
+  
+  ![Opsgenie endpoints](https://dytvr9ot2sszz.cloudfront.net/logz-docs/notification-endpoints/opsgenie-endpoint.png)
```

Comment:
![image](https://user-images.githubusercontent.com/6845404/76897676-d969d500-689c-11ea-9d5d-8565601d7118.png)

This would be fine without the screen shot, and then you don't have to break up the bullet list.

Reply:
@imnotashrimp  - This screenshot was specifically requested by the PM. I could add a third section so as not to break up the list.

Reply:
@shalper Yes, on second read - the bullet list should be reorganized anyway. Testing is part of the creation process - so those two bullets can be grouped together.

Then there'd still be the issue of having the screen shot and a note, one right on top of the other.